File: practica4.cpp
#include <iostream>
using namespace std;
#include <string>

int parentesis(string);

int main(){
  string a ;
  cout<<"Ingresa parentesis: ";
  cin >> a;
  cout<<endl<<a[0];
  int n; 
  n  = parentesis(a)*2;
  cout<<"\nLos caracteres () se repiten \t " << n<< " veces"<<endl ; 
  return 0;
}

int parentesis(string a){
  int n = sizeof(a);
  int j = 0; //contador para los parentesis
  for(int i = 0 ; i < n ; i++  ){
    if( (a[i] == '(' ) && (a[i+1] == ')')){
      j++;
      i++;
    }
  }
  return j;
} 

File: practica4.py
def parentesis(a):
    j = 0
    n = len(a)
    for i in range (n-1):
        if (a[i] == '(') and (a[i+1] == ')'): 
            i = i + 1 
            j = j + 1
    return j              

a = input('Ingresa parentesis (): ')
n = parentesis(a)*2
print('Los () se repiten ', n, 'veces')



